parsing/parser.mly

```diff
@@ -1240,20 +1239,20 @@ rec_module_declarations:
   | rec_module_declarations and_module_declaration
       { let (l, ext) = $1 in ($2 :: l, ext) }
 ;
-rec_module_declaration:
+%inline rec_module_declaration:
```

Comment:
I would rename those `first_rec_module_declaration` and `and_rec_module_declaration`.